src/field_theory/galois.lean

```diff
@@ -112,14 +112,38 @@ end is_galois
 
 end
 
+section is_galois_tower
+
+variables (F K E : Type*) [field F] [field K] [field E] [algebra F K] [algebra F E]
+  [algebra K E] [is_scalar_tower F K E]
+
+lemma is_galois.tower_top_of_is_galois [h : is_galois F E] : is_galois K E :=
+⟨is_separable_tower_top_of_is_separable K h.1, normal.tower_top_of_normal F K E h.2⟩
+
+variables {F E}
+
+@[priority 100] -- see Note [lower instance priority]
+instance is_galois.tower_top_intermediate_field (K : intermediate_field F E) [h : is_galois F E] :
+  is_galois K E := is_galois.tower_top_of_is_galois F K E
+
+lemma is_galois_iff_is_galois_bot : is_galois (⊥ : intermediate_field F E) E ↔ is_galois F E :=
+begin
+  split,
+  { intro h,
+    exactI is_galois.tower_top_of_is_galois (⊥ : intermediate_field F E) F E },
+  { intro h,
```

Comment:
I think this can be `rintor h, apply_instance`.

Reply:
That alone doesn't seem to work. Is there a shorter way to write `intro h, haveI := h, apply_instance`?

Reply:
`introI h, apply_instance` seems to work